src/commands/lightning/dev/app.ts

```diff
@@ -71,20 +55,29 @@ export default class LightningDevApp extends SfCommand<void> {
       summary: messages.getMessage('flags.device-type.summary'),
       char: 't',
       options: [Platform.desktop, Platform.ios, Platform.android] as const,
-      default: Platform.desktop,
     })(),
     'device-id': Flags.string({
       summary: messages.getMessage('flags.device-id.summary'),
       char: 'i',
     }),
   };
 
+  private static async determinePlatform(flags: { 'device-type'?: Platform }): Promise<Platform> {
+    let platform = flags['device-type'];
+
+    if (!platform) {
+      platform = await PromptUtils.promptUserToSelectPlatform();
+    }
+
+    return platform;
```

Comment:
Can this be used instead?

`return platform ?? await PromptUtils.promptUserToSelectPlatform();`